api/routes/api_routes.py

```diff
@@ -125,18 +120,19 @@ def get_mortgage_rate(id):
 @api.route('/api/v1/mortgage-rate', methods=['GET'])
 def all_mortgage_rate():
     return mortgageratecontroller.all_mortgage_rate()
+#
+# @api.route('/api/v1/mortgage-rate', methods=['POST'])
```

Comment:
I see we are commenting these routes until Iteration2 - where we have admin authorization on these routes